packages/typescript-estree/src/ts-estree/ts-nodes.ts

```diff
@@ -2,25 +2,26 @@ import type * as ts from 'typescript';
 
 // Workaround to support new TS version features for consumers on old TS versions
 // Eg: https://github.com/typescript-eslint/typescript-eslint/issues/2388, https://github.com/typescript-eslint/typescript-eslint/issues/2784
-/* eslint-disable @typescript-eslint/no-empty-interface */
+/* eslint-disable @typescript-eslint/ban-ts-comment, @typescript-eslint/prefer-ts-expect-error, @typescript-eslint/no-empty-interface */
 declare module 'typescript' {
```

Comment:
You need to add the two new type names to this augmentation so we can maintain backwards compact with API users not yet on TS 5.3

Reply:
Err, sorry, which ones?

Reply:
Ah, right, `ts.ImportAttribute`, `ts.ImportAttributes`.